openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -258,8 +258,16 @@ def _parent_in_hierarchy(self, placeholder, container):
             return
 
         roots = cmds.sets(container, q=True)
+        ref_node = get_reference_node(roots)
         nodes_to_parent = []
         for root in roots:
+            if ref_node:
+                ref_root = cmds.referenceQuery(root, n=True)[0]
```

Comment:
Please use full name of arguments `nodes=True` instead of `n=True`.
Also, assuming the very first node in a reference is the highest root node seems highly optimistic?

Reply:
> Also, assuming the very first node in a reference is the highest root node seems highly optimistic?

Yeah, I was thinking that but using existing code logic.

Reply:
```suggestion
                ref_root = cmds.referenceQuery(root, nodes=True)[0]
```